layouts/partials/footer.html

```diff
@@ -91,6 +91,9 @@ <h3 class="h6 text-white">Company</h3>
         <ul class="list-group list-group-transparent list-group-white list-group-flush list-group-borderless mb-0">
           <li><a class="list-group-item list-group-item-action"
               href="/press/">Press</a></li>
+          <ul class="list-group list-group-transparent list-group-white list-group-flush list-group-borderless mb-0">
```

Comment:
the forum link isn't indented with the rest of links
![image](https://github.com/simple-login/website/assets/3167780/58549808-237d-47c4-bd9f-449dfd0f4173)

Reply:
I'd also put it in the "Resources" section